### ISSUE TITLE
In testing [delete source link] to see if that section will disappear, the link does not seem to disappear

### ISSUE THREAD

**Issue by stephtngu** [consumer-reports-digital-lab/dark-patterns#65]
wordpress
![image](https://user-images.githubusercontent.com/60265274/118066083-0d8b3980-b36c-11eb-885f-331a8e6f7c1b.png)

website
![image](https://user-images.githubusercontent.com/60265274/118066993-ac646580-b36d-11eb-82fa-c1394c49673f.png)

referencing #28

**Comment by tbeck** [comment]
@stephtngu - We have pushed an update to the logic that fixed this as well as the build error.

https://darkpatterns.gatsbyjs.io/example/forced-me-to-agree-to-personalized-advertising-in-a-popup-to-continue-playing/

I noticed that there are some html tags that we need to adjust styles to account for though. Creating a new issue to track these efforts: #70

**Comment by stephtngu** [comment]
@tbeck I re-added the source link to re-add the section but it is now not rebuilding to revert to the previous state. 
https://darkpatterns.gatsbyjs.io/sightings/forced-me-to-agree-to-personalized-advertising-in-a-popup-to-continue-playing/
![image](https://user-images.githubusercontent.com/60265274/118225593-67b2fa00-b453-11eb-8aaf-fcd58c5f479e.png)

**Comment by tbeck** [comment]
Hi @stephtngu - It looks like we forgot to update the build hook after Ben and I updated the Gatsby Cloud account. So updates to content in WP were not triggering a build. I've changed this and tested on the instance we are using for reference.

In the build history the builds triggered by updates in WP are denoted with "Data Update from Wordpress"

https://www.gatsbyjs.com/dashboard/2d3c1fd2-01f0-4800-9690-2dd6b3747381/sites/0665d476-c80b-44f5-bfb5-0b9ce93ee475/branches/main/builds
